### ISSUE TITLE
[Feature]: Remove the allowlist for allowed domains in READMEs

### ISSUE THREAD

**Issue by JonDouglas** [NuGet/NuGetGallery#10198]
### The Elevator Pitch

The allowlist concept is too much debt and work for the benefit we get out of it. We cannot continue to follow an allowlist for this functionality and should allow people to use it freely as a trusting community and when people violate our terms, we take swift action with regards to TOS.

**Comment by erdembayar** [comment]
Related issue: https://github.com/NuGet/NuGetGallery/issues/8856

**Comment by erdembayar** [comment]
Related issue: https://github.com/NuGet/NuGetGallery/issues/8878

**Comment by erdembayar** [comment]
Related issue: https://github.com/NuGet/NuGetGallery/issues/9960

**Comment by 0xced** [comment]
Yes please! Using https://mermaid.ink would be a great way to workaround #8942 but as the mermaid.ink domain is not in the allowlist I'll have to find another workaround.

**Comment by 0xced** [comment]
@JonDouglas Is there a chance that the allowlist is removed anytime soon?

**Comment by JonDouglas** [comment]
Hey I don't work in NuGet anymore and haven't for a while! But I'll ask around and see if folks can provide an answer.

**Comment by 0xced** [comment]
Thanks! Also, you might want to update your GitHub profile. 😉

**Comment by 0xced** [comment]
@seaniyer @nkolev92 Is there a chance that the allowlist is removed anytime soon?

**Comment by nkolev92** [comment]
Adding @chabiss @lyndaidaii in case they might be able to share something.
